include/pros/adi.hpp

```diff
@@ -39,6 +39,8 @@ class ADIPort {
 	 */
 	ADIPort(std::uint8_t port, adi_port_config_e_t type = E_ADI_TYPE_UNDEFINED);
 
+	virtual ~ADIPort(void);
```

Comment:
You should declare it as `= default`.

Reply:
Is there any difference between doing that and having an empty default implementation?

Reply:
Stackoverflow would seem to indicate that yes, there is a slight difference, so I'll mark is as `default`.